gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/ViewInfo.java

```diff
@@ -144,6 +145,16 @@ ToStringHelper toStringHelper() {
   }
 
   @Override
+  public boolean equals(Object obj) {
+    return obj instanceof ViewInfo && Objects.equals(toPb(), ((ViewInfo) obj).toPb());
```

Comment:
I see no real benefit from renaming `equals` and `hashCode`, do you? There's no good answer to this I guess but I think that defining `equals` and `hashCode` also for abstract classes is a good practice in general. Also this PR is consistent with `JobInfo` hierarchy, so if we change here we must change also there.